packages/twenty-server/src/modules/workflow/workflow-builder/utils/generate-fake-object-record-event.ts

```diff
@@ -2,27 +2,30 @@ import { v4 } from 'uuid';
 
 import { DatabaseEventAction } from 'src/engine/api/graphql/graphql-query-runner/enums/database-event-action';
 import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
+import {
+  Leaf,
+  Node,
+} from 'src/modules/workflow/workflow-builder/types/output-schema.type';
 import { generateFakeObjectRecord } from 'src/modules/workflow/workflow-builder/utils/generate-fake-object-record';
-import { OutputSchema } from 'src/modules/workflow/workflow-builder/types/output-schema.type';
 
 export const generateFakeObjectRecordEvent = (
   objectMetadataEntity: ObjectMetadataEntity,
   action: DatabaseEventAction,
-): OutputSchema => {
+): Record<string, Leaf | Node> => {
   const recordId = v4();
   const userId = v4();
   const workspaceMemberId = v4();
 
   const after = generateFakeObjectRecord(objectMetadataEntity);
   const formattedObjectMetadataEntity = Object.entries(
     objectMetadataEntity,
-  ).reduce((acc: OutputSchema, [key, value]) => {
+  ).reduce((acc: Record<string, Leaf | Node>, [key, value]) => {
     acc[key] = { isLeaf: true, value };
 
     return acc;
   }, {});
 
-  const baseResult: OutputSchema = {
+  const baseResult: Record<string, Leaf | Node> = {
```

Comment:
Maybe we could create an abstract type for `Record<string, Leaf | Node>`? I don't know much about that part of the codebase so I'm unsure why we chose to replace `OutputSchema` with another type. However, I think it would be better not to put `Record<string, Leaf | Node>` everywhere in the app and give it a business name.

Reply:
it will be BaseOutputSchema!